libutils/json.c

```diff
@@ -376,7 +392,12 @@ int JsonCompare(const JsonElement *const a, const JsonElement *const b)
         return JsonContainerCompare(a, b);
 
     case JSON_ELEMENT_TYPE_PRIMITIVE:
-        return strcmp(a->primitive.value, b->primitive.value);
+    {
+        int ret = strcmp(a->primitive.value, b->primitive.value);
```

Comment:
`StringEqual()`

Reply:
cargo cult copy paste, want me to fix the other instances in the file? ;) I think there are some.